tools/build/scripts/rush/audit.js

```diff
@@ -37,11 +37,12 @@ const rushCommonDir = path.join(__dirname, "../../../../common/");
   // for development dependencies only.
   // All security issues should be addressed asap.
   const excludedAdvisories = [
-    1674, // https://npmjs.com/advisories/1674
     1700, // https://npmjs.com/advisories/1700
```

Comment:
@grigasp would you or someone on the team be able to look at fixing this issue?  The path to the issue is `..__..__test-apps__ninezone-test-app>react-markdown>remark-parse>trim`.  I attempted to update to the latest major version of react-markdown and hit some build errors.